### PR TITLE
Sass 872

### PR DESCRIPTION
**Pull request by Dula-J**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [ ]  Have you checked the PR Builder passes?
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: app/views/authErrorPages/AgentAuthErrorPageView.scala.html [hunk 1]
```diff
@@ -28,8 +28,11 @@
         @{messages("error.agent.paragraph2")}</a>
     @{messages("error.agent.paragraph3")}
 </p>
-<p class="govuk-body">@{messages("error.agent.paragraph4")}</p>
-
+<p>
+<a class="govuk-link" href=@appConfig.viewAndChangeEnterUtrUrl target="_blank" rel="noopener noreferrer">
```
Comment (BenCosfordHMRC) [inline app/views/authErrorPages/AgentAuthErrorPageView.scala.html:32]: should it open in a new tab or nah?

File: it/controllers/errors/AgentAuthErrorControllerISpec.scala [hunk 2]
```diff
@@ -49,7 +49,7 @@ class AgentAuthErrorControllerISpec extends IntegrationTest with ViewHelpers {
         titleCheck(ExpectedResults.title)
         h1Check(ExpectedResults.heading,"xl")
         textOnPageCheck(ExpectedResults.youCan, Selectors.youCan)
-        textOnPageCheck(ExpectedResults.tryAnother, Selectors.tryAnother)
+        buttonCheck(ExpectedResults.tryAnother, Selectors.tryAnother, Some(appConfig.viewAndChangeEnterUtrUrl))
```
Comment (joeyymoore) [inline it/controllers/errors/AgentAuthErrorControllerISpec.scala:52]: Same as the others with the url check being a variable not appConfig

**Comment by joeyymoore** [comment]
Screenshot 2021-06-07 at 10 49 22
I think you removed the full stop for employment-fe and income-fe. Needs removing here for consistency, and then tests updated for the change.

File: test/views/authErrorPages/AgentAuthErrorPageViewSpec.scala [hunk 3]
```diff
@@ -48,7 +49,7 @@ class AgentAuthErrorPageViewSpec extends AnyWordSpec with Matchers with GuiceOne
       h1Check(h1Expected, "xl")
       textOnPageCheck(s"$youCannotViewText $authoriseYouAsText $beforeYouCanTryText", p1Selector)
       linkCheck(authoriseYouAsText, authoriseAsAnAgentLinkSelector, authoriseAsAnAgentLink)
-      textOnPageCheck(tryAnotherClientText, p2Selector)
+      buttonCheck(tryAnotherClientText, anotherClientDetailsButtonSelector, Some(mockAppConfig.viewAndChangeEnterUtrUrl))
```
Comment (joeyymoore) [inline test/views/authErrorPages/AgentAuthErrorPageViewSpec.scala:52]: Again url's need changing to check against a val

**Pull request merged**